### ISSUE TITLE
arm64 support

### ISSUE THREAD

**Issue by toblie** [Thom-x/rtsp-snapshot#1]
Hi,
would be great if you could add aarm64 support for your image. I tried to so some changes within in fork, but I didn't get it working...

Thanks for your help

**Comment by Thom-x** [comment]
I don't know what could we do apart from what you already did:

```
ADD https://github.com/just-containers/s6-overlay/releases/download/v1.21.8.0/s6-overlay-aarch64.tar.gz /tmp/
RUN tar xzf /tmp/s6-overlay-aarch64.tar.gz -C / && rm /tmp/s6-overlay-aarch64.tar.gz
```

**Comment by toblie** [comment]
thX for your fast reply. You have to know... I am only the user not pro building own docker images....
So the error which is shown using this forked git URL with docker build command, is still that its not running because the this:
` The requested image's platform (linux/amd64) does not match the detected host                                        platform (linux/arm64/v8) and no specific platform was requested
`

So maybe there is something more needed, than this little change?

**Comment by Thom-x** [comment]
I see, the base image `dseg/mini_httpd-cgi` is not an arm64 compatible image, that's why it's not working.

Could you try with this Dockerfile:

```
FROM arm64v8/alpine
MAINTAINER Daichi Shinozaki <dseg@shield.jp>

RUN \
 apk add --update mini_httpd bash curl \
 &&\
 sed \
  -i /etc/mini_httpd/mini_httpd.conf \
  -e 's/^chroot$/#chroot/' \
  -e 's/^#nochroot$/nochroot/' \
 &&\
 rm -rf /var/cache/apk/*

EXPOSE 80

VOLUME /var/www/localhost/htdocs

MAINTAINER Maugin Thomas

RUN \
 apk add --update ffmpeg coreutils \
 &&\
 rm -rf /var/cache/apk/*

COPY rootfs /

# S6 OVERLAY
ADD https://github.com/just-containers/s6-overlay/releases/download/v1.21.8.0/s6-overlay-amd64.tar.gz /tmp/
RUN tar xzf /tmp/s6-overlay-amd64.tar.gz -C / && rm /tmp/s6-overlay-amd64.tar.gz

ENTRYPOINT ["/init"]
```

**Comment by toblie** [comment]
this works great now, thx for your help!

**Comment by Thom-x** [comment]
Fixed with  cd5c79464f8a3a2478f4c79f7beb7134311a0163
